File: selection_sort.cs
using System;

namespace Sorting
{
    class Selection_Sort
    {
        public int[] selection_sort(int []arr)
        {
            int i,j,k,temp;
            for(i=0;i<arr.Length-1;i++)
            {
                k=i;
               for(j=i+1;j<arr.Length;j++)
               {
                  if(arr[j]<arr[k])
                  {
                      k=j;
                  }
               }
               if(k!=i)
               {
                   temp=arr[i];
                   arr[i]=arr[k];
                   arr[k]=temp;
               }
            }
            return arr;
        }
        static void Main(string []arg)
        {
            Console.WriteLine("Enter the size of the array:");
            int size=Convert.ToInt32(Console.ReadLine());

            int []arr=new int[size];
            Console.WriteLine("Enter the elements in the list");
            for(int i=0;i<size;i++)
            {
                arr[i]=Convert.ToInt32(Console.ReadLine());
            }
            int []brr=new int[size];
            Selection_Sort obj=new Selection_Sort();
            brr=obj.selection_sort(arr);
            for(int i=0;i<size;i++)
            {
                Console.Write(brr[i]+"\t");
            }

        }
    }
}

File: selection_sort.py
def selection_sort(arr):
    for i in range(0,len(arr)-1):
        k=i
        for j in range(i+1,len(arr)):
            if(arr[j]<arr[k]):
                k=j
        
        if(k!=i):
            temp=arr[i]
            arr[i]=arr[k]
            arr[k]=temp
    return arr

def main():
    size=int(input("Enter the size of array:"))
    arr=[]
    print("Enter the elements in the list:")
    for i in range(0,size):
        element=int(input())
        arr.append(element)
    arr=selection_sort(arr)
    print(arr)

if __name__=="__main__":
    main()